analysis/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "analysis"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+ast = { path = "../ast" }
+itertools = "0.10.5"
+log = "0.4.18"
+number = { version = "0.1.0", path = "../number" }
+pretty_assertions = "1.3.0"
+
+[dev-dependencies]
+parser = { path = "../parser" }
```

Comment:
Where is this used?

Reply:
In the tests which compare blocks of texts, this gives you a diff. Will move to dev-deps!